flow/platforms/sky130hd/sky130hd.lyt

```diff
@@ -101,7 +132,72 @@
    <dummy-calls>false</dummy-calls>
    <blank-separator>false</blank-separator>
   </cif>
+  <mag>
+   <lambda>0</lambda>
+   <tech/>
+   <write-timestamp>true</write-timestamp>
+  </mag>
  </writer-options>
+ <d25>
+  <src># Provide z stack information here
+#
+# Each line is one layer. The specification consists of a layer specification, a colon and arguments.
+# The arguments are named (like "x=...") or in serial. Parameters are separated by comma or blanks.
+# Named arguments are:
+#
+#   zstart   The lower z position of the extruded layer in µm
+#   zstop    The upper z position of the extruded layer in µm
+#   height   The height of the extruded layer in µm
+#
+# 'height', 'zstart' and 'zstop' can be used in any combination. If no value is given for 'zstart',
+# the upper level of the previous layer will be used.
+#
+# If a single unnamed parameter is given, it corresponds to 'height'. Two parameters correspond to
+# 'zstart' and 'zstop'.
+#
+# Examples:
+#   1: 0.5 1.5                    # extrude layer 1/0 from 0.5 to 1.5 vertically
+#   1/0: 0.5 1.5                  # same with explicit datatype
+#   1: zstop=1.5, zstart=0.5      # same with named parameters
+#   1: height=1.0, zstop=1.5      # same with z stop minus height
+#   1: 1.0 zstop=1.5              # same with height as unnamed parameter
+#
+# VARIABLES
+#
+# You can declare variables with:
+#   var name = value
+#
+# You can use variables inside numeric expressions.
+# Example:
+#   var hmetal = 0.48
+#   7/0: 0.5 0.5+hmetal*2        # 2x thick metal
+#
+# You cannot use variables inside layer specifications currently.
+#
+# CONDITIONALS
+#
+# You can enable or disable branches of the table using 'if', 'else', 'elseif' and 'end':
+# Example:
+#   var thick_m1 = true
+#   if thickm1
+#     1: 0.5 1.5
+#   else
+#     1: 0.5 1.2
+#   end
+
+</src>
+ </d25>
  <connectivity>
+  <connection>li1,67/44,met1</connection>
+  <connection>met1,68/44,met2</connection>
+  <connection>met2,69/44,met3</connection>
+  <connection>met3,70/44,met4</connection>
+  <connection>met4,71/44,met5</connection>
```

Comment:
Can you please add vias as symbols instead of GDS layer/data?

Reply:
Also do you know if via pin layers count towards connectivity? If so, please add them.

Reply:
Added the vias as symbols. I'm not sure about the via pin layers, I can't find any reference to the layer/data for them in either the magic tech file or sky130 PDK documentation. Also somewhat suspicious is the datatype is not consistent between layers:

```
via.pin - 68/58
via2.pin - 69/58
via3.pin - 70/48
via4.pin - 71/48
```

Reply:
```
<symbols>li1='67/20+67/5+67/16'</symbols>
<symbols>met1='68/20+68/5+68/16'</symbols>
<symbols>met2='69/20+69/5+69/16'</symbols>
<symbols>met3='70/20+70/5+70/16'</symbols>
<symbols>met4='71/20+71/5+71/16'</symbols>
<symbols>met5='72/20-72/15+72/5+72/16'</symbols>
```

I removed the labels (datatype 5), seems wrong to be including them here. Another thing - why do we subtract datatype 5 (short) only from met5? From reading the PDK these are some sort of resistor for metal layers, and if so shouldn't we subtract it from all metal layers?

Reply:
> I'm not sure about the via pin layers, I can't find any reference to the layer/data for them in either the magic tech file or sky130 PDK documentation.

Yes, that's strange. Let's ignore it for now.

> Another thing - why do we subtract datatype 5 (short) only from met5

I'm not sure I follow - there was no connectivity in the master branch version. Your PR is adding it. I'm guessing you're copying this from elsewhere, in which case I don't know what it's supposed to mean or if it's a mistake.